### DISCUSSION TITLE
Maintainer will soon be away / unable to respond

### DISCUSSION THREAD

**Discussion by mara004** [pypdfium2-team/pypdfium2#172] [Announcements]
Just an FYI for any users of this library that the primary maintainer (me) will soon be away or unable to respond.
(My days are very uncertain ATM due to ongoing serious personal trouble and I don't know how things will turn out.)

(Hint for co-maintainers: If any problem arises, take a look at the `devel-new` branch first, it might already be fixed there.
Apart from outdated readme examples and a very suboptimal test suite, you should theoretically be able to merge that branch. I think/hope that it would greatly improve helper code.)

**Comment by mara004** [comment]
@adam-huganir According to plan, I'll be away starting on Wednesday. Might take several weeks or months until I return (edit: and maybe I won't be able or willing to continue this project afterwards).

**Reply by adam-huganir** [reply]
I'll keep an eye on my notifications in case anything comes up

**Reply by mara004** [reply]
Situation turned unexpectedly, I'm back home, at least for the time being :)
